corehq/apps/linked_domain/tests/test_updates.py

```diff
@@ -1,27 +1,38 @@
 from django.test import SimpleTestCase
-from corehq.apps.users.models import UserRole
+
 from corehq.apps.linked_domain.updates import _copy_role_attributes
+from corehq.apps.users.models import UserRole
 
 
 class RoleCopyTests(SimpleTestCase):
     def test_copy_role_attributes_populates_all_attributes(self):
-        source_role = UserRole(
-            name='test',
-            default_landing_page='testpage',
-            is_non_admin_editable=False,
-        )
+        source_role_json = RoleCopyTests._get_source_role_as_json()
```

Comment:
I don't think this is blocking, but this does change the way the test is read. I prefer tests that contain all information necessary to understand them within their bodies. The new helper function hides that information within its body. Given that we're verifying the `name`, `default_landing_page`, and `is_non_admin_editable` properties, this would seem like a more readable test:
```suggestion
        source_role_json = RoleCopyTests._get_source_role_as_json(
		  name='test',
		  default_landing_page='testpage',
		  is_non_admin_editable=False
		)
```

Reply:
Yeah that is fair. I initially didn't even have the helper, and really only added it with the thought of making it as easy as possible for those in the future who may need to update these tests. It also does hide the fields that the test doesn't care about which is nice, so I like your solution which feels like a good middle ground.

Reply:
7a3f0f43913b681e887329c04a0ffc30309ab39b